### ISSUE TITLE
CSP headers instructions incorrect?

### ISSUE THREAD

**Issue by TheNomad11** [xwiki-labs/cryptpad#698]
[According to the wiki](https://github.com/xwiki-labs/cryptpad/wiki/CSP-headers), CryptPad is built with the assumption CSP will be enabled. To enable them, we need to edit the settings. According to the wiki: 

> These settings can be found in your configuration file in the contentSecurity and padContentSecurity sections.

I looked in the settings file cryptpad/config/config.js, but there are no contentSecurity and padContentSecurity sections.

Any help is highly appreciated, thanks!

**Comment by ansuz** [comment]
You're correct that this wiki page is out of date. We're actively working on a better installation guide and will remove these outdated pages in the near future.

For a production installation see the comments in the provided `[cryptpad/config.example.js](https://github.com/xwiki-labs/cryptpad/blob/main/config/config.example.js) and [cryptpad/docs/example.nginx.conf](https://github.com/xwiki-labs/cryptpad/blob/main/docs/example.nginx.conf).

**Comment by TheNomad11** [comment]
Alright but I cannot find any comments on the first file, the other one does not give clear instructions, and I am using Apache anyway.... 

I added a warning to the wiki page: outdated content

**Comment by ansuz** [comment]
I updated a number of pages on our wiki. The [CSP headers](https://github.com/xwiki-labs/cryptpad/wiki/CSP-headers) page now indicates the purpose of CrypitPad's CSP headers and the relevant systems which must be configured for them to be applied as intended.

I've also removed references to a number of third-party configuration files which were added over time, as their presence gave the impression that they were supported or validated as being correct (they were not).

We use NGINX and provide an example configuration file for it which I use on my own personal instance. Unfortunately, I have never seen an Apache config which correctly replicates our use of headers. My best advice is to migrate to using our NGINX config for your instance or to accept that you will not benefit from intended security benefits it implements.
